WordPressKit/PeopleServiceRemote.swift

```diff
@@ -97,6 +97,45 @@ public class PeopleServiceRemote: ServiceRemoteWordPressComREST {
         })
     }
 
+    /// Retrieves the collection of email followers associated to a site.
+    ///
+    /// - Parameters:
+    ///     - siteID: The target site's ID.
+    ///     - page: The page to fetch.
+    ///     - max: The max number of followers to fetch.
+    ///     - success: Closure to be executed on success.
```

Comment:
What do you think about changing the success description to:

```suggestion
    ///     - success: Closure to be executed on success with an array of EmailFollower.
```

Reply:
Good call, that would make it more accurate. I updated it include both the `EmailFollower` array and the bool passed to the closure.